### ISSUE TITLE
Address input field doesn't work on iOS

### ISSUE THREAD

**Issue by maettyhawk** [smeijer/leaflet-geosearch#141]
The address input field is not selectable in any of the browsers on iOS (Safari, FF, Chrome...).

**Comment by smeijer** [comment]
Thanks for reporting. Unfortunately I can't test on iOS, so more information or a PR would be most welcome.

**Comment by maettyhawk** [comment]
Eventhough it might be a hacky fix, it did the trick for me!

**Comment by smeijer** [comment]
So, do I understand correctly that the real error is; that the `input` is unable to receive the `focus`?

**Comment by maettyhawk** [comment]
When you try to tap on the `input`, nothing happens - so yes, `focus` event doesn't get triggered, whereas `click` does. The bug is also reproducable in your examples.

**Comment by phoenixtrigo** [comment]
@smeijer did you discover a fix for that ? can you share it ? The problem still persist...

**Comment by phoenixtrigo** [comment]
I fixed it by move the geosearch div out of the map div.

```
var el = <HTMLElement>document.querySelector('.leaflet-control-geosearch');
if(this._platform.is('ios')){
      document.getElementById('map_content').appendChild(el);
}
```

not the best solution but it fixed the focus issue

**Comment by jongbonga** [comment]
hey, @phoenixtrigo I have the same issue and your workaround is not working for me. do you know of another alternative?

**Comment by phoenixtrigo** [comment]
@jongbonga why it’s not working ? Can you detail more ? Or give a jsfiddle with example ? My code sample are from a ionic (typescript) snippet, not exactly the same as Js

**Comment by jongbonga** [comment]
Also using Ionic. I just used my own input instead

**Comment by RaulCE22** [comment]
Hi guys.

You can try this code into mounted() vue lifecycle 


mounted() {

    this.$nextTick(() => {
      (this.$el.getElementsByClassName(
        "leaflet-control-geosearch"
      )[1] as any).ontouchend = (res: any) => {
        res.target.focus();
      };
    });
  },

I use nextTick because this.$el.getElementsByClassName("leaflet-control-geosearch")[1] is undefined inside only mounted function. 

For me this.$el.getElementsByClassName("leaflet-control-geosearch") is an array of two elements.
Index = 1 is the search input. 

Good look :)

**Comment by roux1max** [comment]
Hi everyone !

First, thanks for the lib it spared me a lot of time!

I also discovered this bug on the Chrome devtool with emulated devices. The fix proposed by @RaulCE22 works for me as well, it seems that for some reason the click on the control is intercepted on mobile devices.

I do not have time right now to investigate but maybe I will come back to it!

**Comment by nash-ye** [comment]
Hello,

This problem is still there. Any effective workarounds?

**Comment by smeijer** [comment]
@nash-ye, does the workaround in #142 work for you?

**Comment by nash-ye** [comment]
> @nash-ye, does the workaround in #142 work for you?

Thank you for your reply. Sorry, I had to use another library. I didn't have much time to try different solutions.

**Comment by smeijer** [comment]
This should be fixed in `3.0.1`
